### ISSUE TITLE
(CI 2022.0 cross-platform) : quelea.properties file isn't sorted anymore

### ISSUE THREAD

**Issue by DeeTigIT** [quelea-projection/Quelea#389]
In the past sorting of the quelea.properties file was added:

 SORT quelea.properties file upon Quelea exit #279 

But in the current CI 2022.0 cross-platform releases this feature seems to be missing/malfunctioning:

![S1](https://user-images.githubusercontent.com/39883796/134958205-06cd9f61-6419-4c2e-a9ed-50f4861c9cec.JPG)

I'm using:
- Quelea CI-RELEASE 2022.0 cross-platform
- OpenJDK11U-jre_x64_windows_hotspot_11.0.12_7.msi (Adoptium Eclipse Temurin)

**Comment by berry120** [comment]
Well spotted!

Technical note: This is because we're using the Apache Commons `SortedProperties` class to sort our properties, and it only overrides `keys()`, not `entrySet()`. This wasn't an issue in Java 8 as the Properties class there used `keys()` in its implementation to write out properties - but the same method in Java 11 uses `entrySet()` instead, hence we lose the ordering.

I've created a PR to fix this in the Apache Commons library (https://github.com/apache/commons-collections/pull/256) - but at the moment I'm not sure whether that'll be accepted or what release it'll be in if it is, so I've duplicated the fixed class into Quelea. We'll thus definitely have a fix in place for 2022.0, and in the future if the Apache Commons PR is accepted then we can revert to using that, and remove the duplicated class in the Quelea codebase.
